### PR TITLE
[Req Assist] Update more reqs names deviation and other improvements

### PR DESCRIPTION
**Pull request by jyejare**
PR includes:
- More Packages in the map of name deviations 
- Manage package is now pinned to the version
- Green Message when reqs are up to date

### CODE REVIEW

File: pytest_plugins/requirements/update_requirements.py [hunk 1]
```diff
@@ -36,6 +36,8 @@ def pytest_report_header(config):
         if config.getoption('update_required_reqs') or config.getoption('update_all_reqs'):
             print('Updating the mandatory requirements on demand ....')
             updater.install_req_deviations()
+    else:
+        print(f'{Colored.GREEN}Mandatory Requirements are up to date!{Colored.RESET}')
```
Comment (ogajduse) [inline pytest_plugins/requirements/update_requirements.py:40]: I feel there is no need to put that much stress on it.
```suggestion
        print(f'{Colored.GREEN}Mandatory Requirements are up to date.{Colored.RESET}')
```

File: pytest_plugins/requirements/update_requirements.py [hunk 2]
```diff
@@ -45,3 +47,5 @@ def pytest_report_header(config):
         if config.getoption('update_all_reqs'):
             print('Updating the optional requirements on demand ....')
             updater.install_opt_deviations()
+    else:
+        print(f'{Colored.GREEN}Optional Requirements are up to date!{Colored.RESET}')
```
Comment (ogajduse) [inline pytest_plugins/requirements/update_requirements.py:51]:
```suggestion
        print(f'{Colored.GREEN}Optional Requirements are up to date.{Colored.RESET}')
```

**Review by ogajduse — COMMENTED** [review]
@jyejare Could you also address #11339 in this PR?

File: pytest_plugins/requirements/update_requirements.py (hunk 1 above)
Comment (jyejare) [reply pytest_plugins/requirements/update_requirements.py:40]: Yes, but sometimes users might think that the warning or confirmation isnt provided this plugin so may be isnt working.

Hence this is just a sign of relief.

**Review by omkarkhatavkar — APPROVED** [review]
ACK, @jyejare CI checks are failing

**Comment by jyejare** [comment]
will look

File: pytest_plugins/requirements/update_requirements.py (hunk 2 above)
Comment (jyejare) [reply pytest_plugins/requirements/update_requirements.py:51]: Not a blocker comment ! Ignoring for now !

**Comment by jyejare** [comment]
@ogajduse I commented on #11339 about that issue itself, please let me know if you have counter comment!

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**